src/vscode-bicep/src/commands/deploy.ts

```diff
@@ -366,12 +365,14 @@ export class DeployCommand implements Command {
       const expiresOnTimestamp = String(accessToken.expiresOnTimestamp);
       const portalUrl = subscription.environment.portalUrl;
 
-      let parametersFileName: string | null;
+      let parametersFileName: string | undefined;
       let updatedDeploymentParameters: BicepUpdatedDeploymentParameter[];
       let parametersFileUpdateOption = ParametersFileUpdateOption.None;
 
-      if (parametersFilePath.endsWith(".bicepparam")) {
-        parametersFileName = null;
+      //if no parameter file or bicepparam file is provided then we don't do the  
```

Comment:
Need to run format on the file or run "npm run lint:fix" from command line to fix these warnings.